internal/bminventory/inventory.go

```diff
@@ -1964,6 +1965,15 @@ func (b *bareMetalInventory) GetPresignedForClusterFiles(ctx context.Context, pa
 		return common.GenerateErrorResponder(err)
 	}
 
+	if params.FileName == "manifests" {
```

Comment:
Maybe this part should move before `else if err = b.checkFileForDownload(ctx, params.ClusterID.String(), params.FileName); err != nil ` ? https://github.com/openshift/assisted-service/pull/215/files#diff-97c2fe56dacc4fee9ab38e4dbd281ab8R1964

Reply:
moved above, but had to modify `checkFileForDownload` to allow d/w the file at any given cluster state - as it shouldn't be impacted by the state of the cluster.